storage-proofs/porep/src/stacked/circuit/params.rs

```diff
@@ -154,8 +154,9 @@ impl<Tree: MerkleTreeTrait, G: 'static + Hasher> Proof<Tree, G> {
         // stores the labels of the challenged column
         let mut column_labels = Vec::new();
 
-        // PrivateInput: challenge index
+        // PublicInput: challenge index
         let challenge_num = uint64::UInt64::alloc(cs.namespace(|| "challenge"), challenge)?;
+        challenge_num.pack_into_input(cs.namespace(|| "challenge input"))?;
```

Comment:
@porcuquine why are you using pack_into_input (?), you should just call inputize like we did on other parts

Reply:
`UInt64` is represented as allocated `Boolean`s, and we need to constrain them to decompose a value passed as `Fr` input. This accomplishes that. If `UInt64` were represented by an allocated `Num`, then we would use `inputize`. It's just one constraint, if that's what you're concerned about.